Cargo.toml

```diff
@@ -28,6 +28,8 @@ bitflags = "^1.2"
 
 [target."cfg(not(target_arch = \"wasm32\"))".dependencies]
 hostname = "^0.3"
+openssl-sys = "*"
+openssl = { version = "*", features = ["vendored"] }
```

Comment:
may we should add some version restrictions?

Reply:
Good catch! Why in the main toml we need openssl?